modules/helm/install_test.go

```diff
@@ -48,6 +46,12 @@ func TestRemoteChartInstall(t *testing.T) {
 		},
 	}
 
+	// Add the stable repo under a random name so as not to touch existing repo configs
+	uniqueName := random.UniqueId()
```

Comment:
NIT: perhaps add some sort of prefix to this random name (e.g., `terratest-`) so it's a bit easier to know where it came from?

Reply:
Updated!